### DISCUSSION TITLE
Registering and Configuring Your Access Gateway through NMS issues - Unable to add H/Wid

### DISCUSSION THREAD

**Discussion by robinge330** [magma/magma#7574] [Q&A] [answered]
I am following instructions as per https://docs.magmacore.org/docs/lte/deploy_config_agw

1. Navigate to your NMS instance, https://your-org.nms.yourdomain.com, and log in with the superuser credentials you provisioned for this organization. If this is a fresh Orchestrator install, you will be prompted to create your first network. Otherwise, select "Create Network" from the network selection icon at the bottom of the left sidebar.

The instance present in https://master.nms.magma330.local/ and https://magma-test.nms.magma330.local/ with user as admin/admin. I donot see any button prompted for create your first network

2. Navigate to "Equipment" on the NMS via the left navigation bar, hit "Add Gateway" on the upper right, and fill out the multi-step modal form. Use the secrets from above for the "Hardware UUID" and "Challenge Key" fields.

There is no screen for equipment.

Please could you guide me the the approach to ad the h/w id and be able to find the networks

[NMSissuelogfiles.txt](https://github.com/magma/magma/files/6658044/NMSissuelogfiles.txt)
<img width="1327" alt="NMS1 5Issue1" src="https://user-images.githubusercontent.com/30368558/122108668-768e2300-cdea-11eb-86c4-e3329a0c8ebb.png">
<img width="1327" alt="NMS1 5Issue2" src="https://user-images.githubusercontent.com/30368558/122112442-d8e92280-cdee-11eb-8725-3ebea19fd3cf.png">

**Comment by robinge330** [comment]
[NMSissuelogfiles.txt](https://github.com/magma/magma/files/6658250/NMSissuelogfiles.txt)

Log attached

**Comment by karthiksubraveti** [comment]
Above screenshots indicate that NMS is unable to reach orc8r. Can you dump all logs using 
`kubectl cluster-info dump --output-directory=<output_dir>` and paste it if possible. Kindly sanitize any PII information prior to uploading the cluster logs. Thanks

**Reply by robinge330** [reply]
Please find attached the details
[cluster-info.zip](https://github.com/magma/magma/files/6663334/cluster-info.zip)

**Comment by robinge330** [comment]
Hi,

Thanks for teh support. I have uploaded the details.

Regards,

**Reply by karthiksubraveti** [reply]
Sorry forgot to include. Please run kubectl for orc8r namespace
kubectl -n <orc8r_namespace> cluster-info dump --output-directory=<output_dir>

**Comment by robinge330** [comment]
Hi, please find attached. Thanks for the support.
[ocr_cluster.zip](https://github.com/magma/magma/files/6670467/ocr_cluster.zip)

**Comment by robinge330** [comment]
Hi,

Thanks for support.
Details are attached.

Regards,

**Comment by robinge330** [comment]
Hi,
 This issue can be closed as on-premise deployment of orchestrator need changed in dns. Refer to [orc8r][nms] Use in-cluster dns for orc8r nginx instead of external dns #7633

**Comment by robinge330** [comment]
Hi,

I need one guidance on the below exception. Please could you guide how to resolve this. This seems to be new in v1.5
<img width="301" alt="Networkissue1" src="https://user-images.githubusercontent.com/30368558/122593622-e93e0f00-d033-11eb-9296-15a7931712ef.PNG">

**Comment by robinge330** [comment, marked as answer]
The error shown is **validation failure list: id in body should match '^[\da-z_-]+$'**
I could not find any documentation of naming convention for Network ID. Please could you guide us.

**Comment by robinge330** [comment]
this can be closed as we undestand now only lowercase names are supported for networkid
